editor/src/components/inspector/sections/layout-section/conditional-section.tsx

```diff
@@ -122,54 +183,65 @@ export const ConditionalSection = React.memo(({ paths }: { paths: ElementPath[]
           <InspectorSectionIcons.Conditionals style={{ width: 16, height: 16 }} />
           <span>Conditional</span>
         </FlexRow>
-        {condition != 'not-overridden' ? (
+        {conditionOverride != 'not-overridden' ? (
           <SquareButton
             highlight
-            onClick={setCondition(null)}
+            onClick={setConditionOverride(null)}
             data-testid={ConditionalsControlSectionCloseTestId}
           >
             <FunctionIcons.Delete />
           </SquareButton>
         ) : (
           <SquareButton
             highlight
-            onClick={setCondition(true)}
+            onClick={setConditionOverride(true)}
             data-testid={ConditionalsControlSectionOpenTestId}
           >
             <Icons.Plus style={{ opacity: 'var(--buttonContentOpacity)' }} />
           </SquareButton>
         )}
       </InspectorSubsectionHeader>
-      {when(
-        condition != 'not-overridden',
+      {conditionExpression !== 'multiselect' ? (
```

Comment:
probably good to use `when` as well (or `unless` if you want to keep the comparison with `"multiselect"`)